lib/crawler_web/controllers/keyword_controller.ex

```diff
@@ -24,18 +24,16 @@ defmodule CrawlerWeb.KeywordController do
   end
 
   defp process_uploaded_file(conn, changes) do
-    case CSVParser.parse(changes.file.path) do
-      {:ok, keyword_list} ->
-        {num_keyword, _list} =
-          Keywords.create_keyword_list(keyword_list, conn.assigns.current_user.id)
-
-        conn
-        |> put_flash(
-          :info,
-          gettext("%{num_keyword} keywords were uploaded!", num_keyword: num_keyword)
-        )
-        |> redirect(to: Routes.home_path(conn, :index))
-
+    with {:ok, keyword_list} <- CSVParser.parse(changes.file.path),
+         {:ok, keyword_ids} <-
+           Keywords.create_keyword_list(keyword_list, conn.assigns.current_user.id) do
+      conn
+      |> put_flash(
+        :info,
+        gettext("%{num_keyword} keywords were uploaded!", num_keyword: length(keyword_ids))
```

Comment:
"1 keywords were uploaded!" is possible message with this approach :slightly_smiling_face:

Reply:
Yeah, I'm aware of it too 🤣 but it's not a big deal, just an "English" thing. Could we leave it for now?

Reply:
Yep, it is minor thing and we can leave it as is, I just wanted to point that in client project there would be good to use gettext plural approach